tests/ffi/tcp_max_syn_backlog/README.md

```diff
@@ -0,0 +1,17 @@
+Title:
+    Validate if tcp_max_syn_backlog network parameter on Host can be resilient against random changes of this parameter in QM.
+
+Description:
+    The purpose of this test script is to validate if tcp_max_syn_backlog parameter value on Host, can be resilient against random changes of this parameter in QM. This is validated by creating a test environment, that generates and sets a random tcp_max_syn_backlog value in range from 128 to 1024 in QM and as soon as QM gets a new value, the test verifies that tcp_max_syn_backlog parameter on Host stays the same and not changes because of the QM network changes.
+
+Input:
+    Environment variables:
```

Comment:
If you are not using env variables, I think just set this as blank:

Input:

Reply:
Cleared up and removed.